tests/test-war-smoke/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM ${jetty.test.image}
+ADD root /app
```

Comment:
Pushed, though should axe the commented out plugin in the test-war-smoke/pom.xml eventually, left as placeholder for unpack approach for time being.

going to / will just show a simple html file now, simple test is run against hello servlet which can all be expanded on quickly once the basic approach is deemed acceptable